pkg/flowcli/util/utilities.go

```diff
@@ -113,3 +113,12 @@ func ContainsString(s []string, e string) bool {
 	}
 	return false
 }
+
+func ValidAddress(value string) bool {
```

Comment:
```suggestion
func ParseAddress(value string) (flow.Address, bool)
```

```go
address, isValid := util.ParseAddress(raw)
if !isValid {
  ...
}
```

Just a suggestion, your way works too